metadata-ingestion/src/datahub/ingestion/source/unity/proxy_profiling.py

```diff
@@ -152,21 +161,47 @@ def _check_analyze_table_statement_status(
                 statement_id
             )
             self._raise_if_error(response, "get-statement")
-            status = response.status
+            status = response.status  # type: ignore
```

Comment:
What's the issue here?

Reply:
execute_response.status  is nullable however, we already check at the top whether status is present. If it's present in one response, it should also be present in subsequent responses ideally. But mypy does not handle that, as we reset response in every iteration.